cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -366,8 +366,8 @@ func ValidateHostPort(endpoint string, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
-// ValidateIPNetFromString validates network portion of ip address
```

Comment:
maybe you can keep this function and continue using it for validation of the string